Content.Server/GameObjects/Components/Power/PowerTransferComponent.cs

```diff
@@ -140,17 +141,16 @@ public bool CanConnectTo()
 
         public bool AttackBy(AttackByEventArgs eventArgs)
```

Comment:
Any reason this isn't using `IWirecutterAct`?

Reply:
I removed ToolActs now.